weibo/spiders/search.py

```diff
@@ -354,7 +354,7 @@ def parse_weibo(self, response):
                 weibo = WeiboItem()
                 weibo['id'] = sel.xpath('@mid').extract_first()
                 weibo['bid'] = sel.xpath(
-                    './/p[@class="from"]/a[1]/@href').extract_first(
```

Comment:
this "p" label works on my computer, but "div" does not, so it depends on...

Reply:
I am also confused, maybe it is the problem of Weibo. May I ask the parameters in your settings.py.

Reply:
[#254](https://github.com/dataabc/weibo-search/issues/254#issuecomment-1245660105), I guess it should be the cause of the cookie.

> this "p" label works on my computer, but "div" does not, so it depends on...